### PR TITLE
Add language-html-angular support

### PR DESCRIPTION
**Pull request by jakewtaylor**
Adds in syntax highlighting for the [language-html-angular](https://atom.io/packages/language-html-angular) package.

### CODE REVIEW

File: styles/syntax/ng.less [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    color: @mono-1;
+  }
+
+  &.syntax--tag {
+    color: @hue-1;
+  }
+
+  &.syntax--attribute-with-value {
+    .syntax--attribute-name {
+      color: @hue-6-2 !important;
```
Comment (simurai) [inline styles/syntax/ng.less]: Is `!important` needed here? I tested it with this [example](https://github.com/drootz/language-html-angular/blob/f3a36caf20ef64125adc418f1da57414b35fafd0/example/example.html) and haven't seen a difference.

To increase specificity, `.syntax--attribute-name` could also be repeated, like 

```
.syntax--attribute-name.syntax--attribute-name {
```

Then it's still possible to use `!important` in user's styles.less file to override it.

**Comment by simurai** [comment]
Here a before/after gif with [this example](https://github.com/drootz/language-html-angular/blob/f3a36caf20ef64125adc418f1da57414b35fafd0/example/example.html):

ng

File: styles/syntax/ng.less (hunk 1 above)
Comment (jakewtaylor) [reply styles/syntax/ng.less]: Yep, not sure why I had that there, works fine without.

**Comment by jakewtaylor** [comment]
@simurai, I've made the changes - waiting for review 😄

**Pull request merged**